Tasks/Common/kubernetes-common-v2/funckubernetesutility.ts

```diff
@@ -0,0 +1,30 @@
+import * as tl from "azure-pipelines-task-lib/task";
+import * as tr from "azure-pipelines-task-lib/toolrunner";
+
+export function getFuncDeployCommand(funcToolsPath: string, secretName: string, appName: string, namespace: string, imageName: string, registry: string, pullSecretName: string, args: string): tr.ToolRunner {
```

Comment:
You could just exec here itself, right?

Reply:
Keeping the execution part in the task code as I am following a pattern of building the command, then calling execCommand. Changing this would require multiple changes with no apparent benefit. We can discuss this later.